debian/cloudstack-management.postinst

```diff
@@ -65,4 +65,9 @@ if [ "$1" = configure ]; then
     grep -s -q "db.simulator.driver=jdbc:mysql" ${CONFDIR}/${DBPROPS} || sed -i -e "\$adb.simulator.driver=jdbc:mysql" ${CONFDIR}/${DBPROPS}
 fi
 
+    # Create symbolic link for python
+    if [ ! -f "/usr/bin/python" ]; then
+        ln -s /usr/bin/python2 /usr/bin/python
+    fi
```

Comment:
Same as on the mgmt server

Reply:
@wido this is a patch for 4.14.0.0
if we change all python2 scripts to python3 in 4.15, we definitely do not need it.

Reply:
@weizhouapache As the PR is against master it's for 4.15, therefor I assumed that.

Py3 support is finished for the hypervisors already by @GabrielBrascher , that all works.

No more need for Py2 on hypervisors.

Reply:
@wido cool. do you have more changes than security_group.py ? could you create a PR ?
there are still some scripts in 4.14 which are written with python2 and used on kvm hosts, for example all scripts in agent/bindir/

Reply:
@weizhouapache PR #3730 updates VR's scripts from python2 to python3. So far it has been tested on Advanced network with Security Group. We should have all python2 code migrated to python3 in 4.15.0.0; as soon as 4.14 is released and the master branch is "unfreeze", we will get back on porting python2 to python3.

Additionally, PR #3730 will be coordinated with other PRs (e.g. #3797). Any help is welcome :slightly_smiling_face: